modules/travel_pay/app/services/travel_pay/claims_service.rb

```diff
@@ -2,7 +2,13 @@
 
 module TravelPay
   class ClaimsService
-    def get_claims(veis_token, btsss_token, params = {})
+    def initialize(travel_pay_session)
+      @session = travel_pay_session
+    end
+
+    # def get_claims(veis_token, btsss_token, params = {})
+    def get_claims(params = {})
+      @session.get_tokens => { veis_token:, btsss_token: }
```

Comment:
So this might be nit-picky, but to me an "authenticated session" would mean the session has already been authenticated _by all required_ parties, so the "session", to my mind, would already have called the `get_tokens` endpoint and includes the tokens in the existing session.... the way things are set up on the API side it might not work that way, but this is confusing to me when I think about an "authenticated session".... 🤔

Reply:
I like this comment. What other names are you thinking?